packages/react-slider/src/components/Slider/stories/SliderControlled.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { Label } from '@fluentui/react-label';
+import { Slider } from '../../../index'; // codesandbox-dependency: @fluentui/react-slider ^9.0.0-beta
```

Comment:
```suggestion
import { Slider } from '../../../index';
```

These are no longer necessary